### ISSUE TITLE
NVDA doesn't read "/" in closing tag of html in chrome dom

### ISSUE THREAD

**Issue by gauravahir28** [nvaccess/nvda#11763]
### Steps to reproduce:
Open Any website in chrome.
Inspect any element.
Navigate through dom.
### Actual behavior:
Suppose following is the code you encounter while navigating dom:
`<div></div>`
NVDA Reads `<div>` for both the opening/closing tag.
### Expected behavior:
NVDA should read `</div>` 
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
20.3
#### Windows version:
Windows 10 (2004) 
#### Name and version of other software in use when reproducing the issue:
Chrome (86)
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Works with older versions
#### If addons are disabled, is your problem still occuring?
Not tried
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
Yes

**Comment by gauravahir28** [comment]
I have found same issue on MS Edge chromioum (86)

**Comment by gauravahir28** [comment]
This seems to be fixed in chrome 93 and above so closing it.
